src/main/java/com/google/sps/tool/EntityTestingTool.java

```diff
@@ -41,6 +41,16 @@ public static boolean checkForNoNullValues(Entity card) {
     return folders;
   }
 
+  public static Folder populateDatastoreWithAFolder(Folder FOLDER_A, DatastoreService datastore, String USERKEY) {
```

Comment:
why USERKEY? I have seen this in other files as userKey.

Reply:
It's needed in order to set an ancestry path from the User --> Folder. I'm just simulating what a Folder Entity would have, which is being keyed by the user's key.